src/services/blocks/BlocksService.ts

```diff
@@ -496,13 +507,13 @@ export class BlocksService extends AbstractService {
 		}
 
 		const multiplier =
-			await api.query.transactionPayment?.nextFeeMultiplier?.at(parentHash);
+			await historicApi.query.transactionPayment?.nextFeeMultiplier();
```

Comment:
From what I understand we may actually need `(await api.at(parentHash)).query.transactionPayment?.nextFeeMultiplier()`